waffle_configs/bedrock-prod.env

```diff
@@ -27,3 +27,4 @@ SWITCH_FIREFOX_YANDEX=on
 SWITCH_FIRSTRUN_COPY_EXPERIMENT=off
 SWITCH_NEWSLETTER_MAINTENANCE_MODE=off
 SWITCH_SHOW_POCKET_FEED=on
+SWITCH_TRANSPARENCY_REPORT=off
```

Comment:
The value should be `on` (for stage as well). But we won't merge this until Dec 27 so no rush.